modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberPopupTest.ts

```diff
@@ -19,6 +19,20 @@ describe('browser.tinymce.themes.silver.throbber.ThrobberPopupTest', () => {
       ed.ui.registry.addContextMenu('test', {
         update: Fun.constant('test-item')
       });
+      ed.ui.registry.addContextToolbar('test-2', {
+        items: 'bold | italic',
+        scope: 'node',
+        position: 'selection',
+        predicate: (node: Node) => {
+          if (node.nodeType === NodeTypes.ELEMENT) {
+            const elem = node as HTMLElement;
+            if (elem.classList.contains('ctx-menu-me')) {
+              return true;
+            }
+          }
+          return false;
```

Comment:
```suggestion
          } else {
            return false;
          }
```
This is specific "if do this this else do that" logic so we should be capturing that.